rpc/jsoncodec/doc.go

```diff
@@ -1,5 +1,13 @@
-// Copyright 2022 Canonical Ltd.
+// Copyright 2025 Canonical Ltd.
 // Licensed under the AGPLv3, see LICENCE file for details.
```

Comment:
After discussing with Heather, it looks like this doc would be fine with just this:

> Package jsoncodec defines the methods for reading and writing RPC messages (in JSON format) and the connection (gorilla websocket) which passes them.

Reply:
Thanks for updating! According to my chat with Heather, you can delete the list that follows as well.